stdlib/public/core/ArrayType.swift

```diff
@@ -17,13 +17,13 @@ internal protocol ArrayProtocol
 {
   //===--- public interface -----------------------------------------------===//
   /// The number of elements the Array stores.
-  var count: Int { get }
+  override var count: Int { get }
```

Comment:
Did you add these? Or were they always there, previously unnecessarily?

Reply:
then I suspect these are a weird old anomaly we can remove. Will put up a PR.

Reply:
(after you land this one to avoid conflicting you)

Reply:
Up to you! Once this lands, it's no longer an ABI change to remove it :)